pkg/apis/kops/validation/validation.go

```diff
@@ -290,17 +291,21 @@ func validateClusterSpec(spec *kops.ClusterSpec, c *kops.Cluster, fieldPath *fie
 }
 
 type cloudProviderConstraints struct {
-	requiresSubnets      bool
-	requiresNetworkCIDR  bool
-	prohibitsNetworkCIDR bool
-	requiresSubnetCIDR   bool
+	requiresSubnets                                 bool
+	requiresNetworkCIDR                             bool
+	prohibitsNetworkCIDR                            bool
+	requiresNonMasqueradeCIDR                       bool
+	requiresServiceClusterSubnetOfNonMasqueradeCIDR bool
```

Comment:
I thought this was the case everywhere (per discussion in other PR), but maybe it isn't?

(Also, naming nit - maybe we should call it ServiceCIDR instead of ServiceClusterSubnet, because I don't think anyone actually allocates a cloud "subnetwork" to it.  But maybe this is where I'm getting confused!)

Reply:
The current semantics are that GCE doesn't require this. If we bring GCE in line with other cloud providers then we can remove this exception.

The reading was intended to be "the cloud provider _requires_ the _SericeCluster_ be a _subnet_ of the _nonMasquradeCIDR_".